<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelPage</class>
 <widget class="QWidget" name="ChannelPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>334</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>     
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="generalGroupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="emoteicon_checkBox">
        <property name="text">
         <string>Load Emoticons</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="tabsGroupBox">
     <property name="title">
      <string>Tabs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="GroupFrameSettingsWidget" name="groupFrameSettingsWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="downloadsGroupBox">
     <property name="title">
      <string>Downloads</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QSpinBox" name="autoDownloadSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Maximum Auto Download Size (in GBs)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>     
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupFrameSettingsWidget</class>
   <extends>QWidget</extends>
   <header>gui/settings/GroupFrameSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
